lib/engine/game/g_18_sj/game.rb

```diff
@@ -180,9 +184,9 @@ class Game < Game::Base
              260
              280
              300],
-          %w[46y 55y 60y 65 70 76 82p 90 100 111 125 140 155 170 185 200],
-          %w[39o 48y 54y 60y 66 71 76p 82 90 100 110 120 130],
-          %w[32o 41o 48y 55y 62 67 71p 76 82 90 100],
+          %w[46y 55y 60y 65 70 76 82pm 90 100 111 125 140 155 170 185 200],
+          %w[39o 48y 54y 60y 66 71 76p 82m 90 100 110 120 130],
+          %w[32o 41o 48y 55y 62 67 71p 76 82m 90 100],
```

Comment:
Why are the other rows spread over x lines instead?

Reply:
Fixed now